odoo/openupgrade/deferred_100.py

```diff
@@ -0,0 +1,17 @@
+# coding: utf-8
```

Comment:
Just a matter of getting used to ;-) It *is* consistent with previous iterations.